lib/course_planner/permissions.ex

```diff
@@ -39,6 +41,9 @@ defimpl Canada.Can, for: CoursePlanner.User do
   def can?(%User{role: role}, _action, %User{})
     when role in ["Teacher", "Student", "Volunteer"], do: false
 
+  def can?(%User{role: "Coordinator"}, _action, SettingController), do: true
```

Comment:
Coordinator is already has the right in the following line "def can?(%User{role: "Coordinator"}, _, _), do: true"